augur/cli/util.py

```diff
@@ -52,9 +53,28 @@ def cli_kill_processes():
     if processes != []:
```

Comment:
see my note under kill_processes ... I am not exactly sure why we have two nearly identical commands, but I suspect its backward compatibility related. :)

Reply:
The `cli_kill_processes` really needs to be changed to just be a wrapper for the actual `kill_processes` function. There are two version because `kill_processes` needs to be imported elsewhere in the code, and with how Augur is set up we can't use the `cli_kill_processes` command to run it has to be repeated. This will be fixed later but this is why there are 2 versions.

Reply:
OK ... we can do this in the next release.